newIDE/app/src/Utils/GDevelopServices/Authentication.js

```diff
@@ -140,6 +142,8 @@ export default class Authentication {
   getFirebaseUser = (cb: (any, ?FirebaseUser) => void) => {
     if (!this.isAuthenticated()) return cb({ unauthenticated: true });
 
+    this.auth.currentUser.reload();
```

Comment:
i'm wondering if the reload should happen in this function (which is called in multiple places) or in another function, like `fetchFirebaseUser`

Reply:
I think we can leave it here. the function `getFirebaseUser` is used in only 1 place, when the full profile info is fetched. I think it makes sense to reload the data from Firebase too at that time to make sure we've got correct information.